sixth.py

```diff
@@ -0,0 +1,16 @@
+
+string = "PQRQRQRQRQRQ" 
+iter(string)
+substring = "QRQ"
+
+
+total_str_length=len(string)
+total_sub_str_length = len(substring)
+
+cnt = 0
+for i in range(0,(total_str_length-2)):
+    tmp = string[i:i+total_sub_str_length]
+    if tmp == substring:
+        cnt+=1
+    
+print(cnt)
```

Comment:
Run your code for the following inputs
string = "PQRQRQRQRQRQA"
substring = "QA"